docs/overrides/home.html

```diff
@@ -0,0 +1,235 @@
+            for multimodal retrieval and generation workflows.
+          </p>
+          <a href="guide/blob" class="md-button">Learn More</a>
+        </div>
+        <div class="lance-feature-demo">
+          <div id="termynal-multimodal"
+            data-termynal=""
+            data-ty-startdelay="2500"
+            data-ty-typedelay="40"
+            data-ty-linedelay="700"
+            style="width: 500px;">
+            <span data-ty="input" data-ty-prompt=">>>">import lance</span>
+            <span data-ty="input" data-ty-prompt=">>>">import pyarrow as pa</span>
+            <span data-ty="input" data-ty-prompt=">>>">schema = pa.schema([</span>
+            <span data-ty="input" data-ty-prompt="...">  pa.field("video", pa.large_binary(),</span>
+            <span data-ty="input" data-ty-prompt="...">    metadata={"lance-encoding:blob": "true"}),</span>
+            <span data-ty="input" data-ty-prompt="...">  pa.field("embedding", pa.list_(pa.float32(), 128))])</span>
+            <span data-ty="input" data-ty-prompt=">>>">lance.write_dataset(table, "multimodal.lance", schema=schema)</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- Feature 4: Data Evolution -->
+  <section class="lance-feature-section reverse">
+    <div class="container">
+      <div class="lance-feature-content">
+        <div class="lance-feature-text">
+          <h2>Data Evolution > Schema Evolution</h2>
+          <p>
+            Schema evolution in most open table formats are metadata only and fast.
+            But when trying to backfill column values in existing rows, a full table rewrite is typically required.
+            Lance supports efficient schema evolution with backfill, making it perfect for ML
+            feature engineering, embedding and media content management.
+          </p>
+          <p>
+            Adding a new column with data is as simple as writing new Lance files to the Lance table -
+            no need to rewrite your entire dataset.
+          </p>
+          <a href="guide/data_evolution" class="md-button">Learn More</a>
+        </div>
+        <div class="lance-feature-demo">
+          <div id="termynal-evolution"
+            data-termynal=""
+            data-ty-startdelay="3500"
+            data-ty-typedelay="40"
+            data-ty-linedelay="700"
+            style="width: 500px;">
+            <span data-ty="input" data-ty-prompt=">>>">import lance</span>
+            <span data-ty="input" data-ty-prompt=">>>">dataset = lance.dataset("my_data.lance")</span>
+            <span data-ty="input" data-ty-prompt=">>>">@lance.batch_udf()</span>
+            <span data-ty="input" data-ty-prompt="...">def add_embeddings(batch):</span>
+            <span data-ty="input" data-ty-prompt="...">  vectors = model.encode(batch["text"])</span>
+            <span data-ty="input" data-ty-prompt="...">  return {"embedding": vectors}</span>
```

Comment:
The two spaces are collapsed to one space here due to HTML rules, making it hard to see the indentation. Maybe use `<pre>` or `<style="white-space: pre;">`